scripts/compare_perf.py

```diff
@@ -37,6 +66,12 @@ def main() -> None:
         total_baseline += baseline_time
         total_latest += latest_time
 
+        # Send PR comment if performance is slower but not past blocking threshold
+        if latest_time < baseline_time * 1.1 or latest_time - baseline_time < 3.5:
+            send_comment(
+                "Potential non-blocking slowdown", github_token, pull_request_number
+            )
```

Comment:
1. I would include the numbers in the comment: baseline and latest times in seconds, percentage difference, reason why the warning is triggered (percentage or absolute difference?), and a summary of the protocol (what did we run and compare against?).
2. 10% slowdown seems reasonable for a warning. Maybe 5% is fine too, since it's only a comment.

If you're not sure, @emjin or other PA dev can do it later.

Reply:
Yeah makes sense to be more clear in the error than just "there was a slowdown" i'll update

Reply:
Updated it to 7%.

For (1) this script doesn't actually have the information on what protocol is being compared. Probably fine for now as that change will need to update how the timing files are generated